front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -32,6 +34,33 @@ const SimulationReportSheet = ({
     path_number2: 'n°YYYYYY',
   };
 
+  const [mapBlob, setMapBlob] = useState<string | null>(null);
```

Comment:
Nit: maybe `mapImageUrl` would be a more appropriate name now that this isn't a `Blob` anymore?

Reply:
of course